config/samples/php7_protobuf_c_example_loadtest.yaml

```diff
@@ -0,0 +1,117 @@
+# Load test configurations generated from a template by loadtest_config.py.
+# See documentation below:
+# https://github.com/grpc/grpc/blob/master/tools/run_tests/performance/README.md#grpc-oss-benchmarks
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  annotations:
+    scenario: php7_protobuf_c_extension_to_cpp_protobuf_sync_unary_ping_pong
+    uniquifier: basic
+  labels:
+    language: php7_protobuf_c
+    prefix: examples
+  name: examples-php7-protobuf-c-php-extension-to-cpp-protobuf-sync-unary-ping-pong-basic
+spec:
+  clients:
+  - build:
+      command:
+      - bash
+      - /build_scripts/build_qps_worker.sh
+    clone:
+      gitRef: master
+      repo: https://github.com/grpc/grpc.git
+    language: php7_protobuf_c
+    name: '0'
+    run:
+      args:
+      - -c
+      - |
+        timeout --kill-after="${KILL_AFTER}" "${POD_TIMEOUT}" \
+        /run_scripts/run_protobuf_c_worker.sh
+      command:
+      - bash
+  driver:
+    language: cxx
+    name: '0'
+    run: { }
```

Comment:
I don't think this should have a space. Was it generated from grpc/grpc? In any case, it should have the same format as the others.

Reply:
Fixed :)